Client/game_sa/premake5.lua

```diff
@@ -16,21 +16,20 @@ project "Game SA"
 	filter "system:windows"
 		includedirs { "../../vendor/sparsehash/current/src/windows" }
 	
-	filter {}
```

Comment:
`filter {}` must be there to reset the filter. Otherwise, `includedirs {"..sdk", ...` will only be added on Windows.